.eslintrc.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  "plugins": ["jest", "@typescript-eslint"],
+  "extends": ["plugin:github/recommended", "plugin:@typescript-eslint/recommended-requiring-type-checking"],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 9,
+    "sourceType": "module",
+    "project": {
+      "project": true,
+      "tsconfigRootDir": __dirname
```

Comment:
Recommended by https://typescript-eslint.io/linting/typed-linting and https://typescript-eslint.io/linting/troubleshooting/#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file